messages/src/lib.rs

```diff
@@ -216,25 +276,83 @@ impl Decode for Duration {
 
 impl Display for Duration {
     fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
-        write!(f, "{} seconds", self.0)
+        write!(f, "{} time_precision units", self.0)
     }
 }
 
-/// DAP protocol message representing an instant in time with a resolution of seconds.
+/// DAP protocol message representing an instant in time as a multiple of the task's time
```

Comment:
As with `Duration`, it's not quite right to call this a multiple.

Reply:
in dd359ca08df24fbbfabe59bb509f892e646aab53